src/plugins/curator/screens/search-teasers/delegate.js

```diff
@@ -10,9 +10,13 @@ class Delegate extends AbstractDelegate {
   }
 
   getSearchParams() {
-    const { statuses } = this.component.props;
+    const { searchNodesRequestState: { request }, statuses } = this.component.props;
 
-    return { sort: SearchTeasersSort.ORDER_DATE_DESC.getValue(), statuses };
+    return {
+      sort: SearchTeasersSort.ORDER_DATE_DESC.getValue(),
```

Comment:
@hunyoboy shouldn't sort come from the request too if request is populated?

Reply:
i agree.. thank you